find-updates.py

```diff
@@ -27,6 +27,21 @@ def write_file(content, filename):
     with open(filename, "w") as fd:
         fd.write(content)
 
+def set_env_and_output(name, value):
+    """
+    helper function to echo a key/value pair to the environement file
```

Comment:
Just noticed this while looking for something else: environement -> environment.